src/main/java/ooss/Teacher.java

```diff
@@ -1,4 +1,42 @@
 package ooss;
 
-public class Teacher {
+public class Teacher extends Person{
+
+    private int id;
+    private String name;
+    private int age;
+    private Klass klass;
+
+    public Teacher (int id, String name, int age){
+        super(id,name,age);
+        this.id = id;
+        this.name = name;
+        this.age = age;
+    }
+
+    @Override
+    public String introduce() {
+        if (klass != null) {
```

Comment:
can store my name is ... that string first then set condition for seeing klass is null or not :)